src/align.cpp

```diff
@@ -1418,18 +1418,19 @@ static CmtAlignType get_comment_align_type(chunk_t *cmt)
 chunk_t *align_trailing_comments(chunk_t *start)
 {
    LOG_FUNC_ENTRY();
-   int          min_col  = 0;
-   int          min_orig = -1;
+   size_t       min_col  = 0;
+   size_t       min_orig = 0;
    chunk_t      *pc      = start;
-   int          nl_count = 0;
+   size_t       nl_count = 0;
    ChunkStack   cs;
    CmtAlignType cmt_type_start, cmt_type_cur;
-   int          col;
-   int          intended_col = cpd.settings[UO_align_right_cmt_at_col].n;
+   size_t       col;
+   size_t       intended_col = cpd.settings[UO_align_right_cmt_at_col].n;
```

Comment:
It is still an interim.
I plan to introduced a variant, such as ```cpd.settings[UO_xyz].u``` for the options, which values are not negativ.
The work is soooo big as I only want to make little changes each PR.
My test reference are all the tests we have today.
It might be necessary to create new tests. Please create "many" PR for that, it will be fine.